packages/overdrive/lib/components/Tabs/Tab.tsx

```diff
@@ -78,12 +92,19 @@ export const Tab = forwardRef<HTMLDivElement, Props>(
 			<>
 				<span className={styles.item}>{children}</span>
 				{typeof indication === 'number' && (
-					<span
-						className={clsx(styles.indication.default, {
-							[styles.indication.active]: isActive,
-						})}>
+					<Text
```

Comment:
The label is too dark.

|CURRENT|NEW|
|--|--|
|![image](https://user-images.githubusercontent.com/599459/88248411-74a80680-cce4-11ea-9407-773962852b6f.png)|![image](https://user-images.githubusercontent.com/599459/88248396-6b1e9e80-cce4-11ea-85b3-47c3f558368f.png)|